libs/sdk-model/src/execution/measure/index.ts

```diff
@@ -345,6 +346,18 @@ export function isArithmeticMeasure(obj: unknown): obj is IMeasure<IArithmeticMe
     return isMeasure(obj) && isArithmeticMeasureDefinition(obj.measure.definition);
 }
 
+/**
+ * Type guard for checking whether an object is a virtual arithmetic measure based on its local ID.
```

Comment:
Please, update this comment to match the new functionality.

Reply:
Updated, thank you!